Support/Image.swift

```diff
@@ -0,0 +1,91 @@
+// Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+import TensorFlow
+
+public struct Image {
+    public enum ByteOrdering {
+        case bgr
+        case rgb
+    }
+
+    enum ImageTensorFormat {
```

Comment:
```suggestion
    enum ImageTensor {
```

I think this should be renamed to `ImageTensor` because it is semantically just a tensor except that the value depends on a format tag. It itself isn't the format -- only the enum case tag is.